.github/CODEOWNERS

```diff
@@ -11,6 +11,9 @@
 #   (See oppia/#10250 for an example.) Please make sure to restore ownership after
 #   the above date passes.
 
+# GitHub Actions & CI workflows.
+.github/*.yaml @BenHenning
```

Comment:
Does `.yaml` and `.yml` works both with this?

Reply:
+1 same question, we should probably add .yml if it doesn't

Reply:
Good point! Added.